File: forJava/src/src/jp/nyatla/nymmd/IMmdPmdRender.java
package jp.nyatla.nymmd;


import jp.nyatla.nymmd.types.Matrix;

public interface IMmdPmdRender
{
	public void dispose();
	public void setPmd(MmdPmdModel i_pmd, MmdDataIo i_io) throws MmdException;
	public void updateSkinning(Matrix[] i_skinning_mat);
	public void render();
}

File: forCS/FW2.0/NyMmd/jp/nyatla/nymmd/cs/IMmdPmdRender.cs
using System;
using System.Collections.Generic;
using System.Text;
using jp.nyatla.nymmd.cs.types;

namespace jp.nyatla.nymmd.cs
{
    public interface IMmdPmdRender:IDisposable
    {
        void setPmd(MmdPmdModel i_pmd, IMmdDataIo i_io);
        void updateSkinning(MmdMatrix[] i_skinning_mat);
        void render();
    }
}
